src/schema/nmdc.yaml

```diff
@@ -223,7 +226,7 @@ classes:
       - extraction_method
       - extraction_target
       - input_mass
-      - quality_control_report
+      - volume
```

Comment:
this doesn't' really have anything to do with QC reports does it? I wonder if I let it sneak in?

Reply:
@turbomam  I think this is suppose to be here. You can see commits to main from a month or two ago from @anastasiyaprymolenna  and you. See https://github.com/microbiomedata/nmdc-schema/commit/4448e41336bede06541bd3c3d2457aec6314f6ab
the slot for Class Extraction seems to have fallen out but the slot usage is there so I think this slot should be on this class. @anastasiyaprymolenna  please confirm.